### PR TITLE
Setup/mongo db

### PR DESCRIPTION
**Pull request by majdya**

### CODE REVIEW

File: mongodb.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// CRUD
```
Comment (jema28) [inline mongodb.js]: Delete this comment 🐱

File: mongodb.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+const { MongoClient } = require("mongodb");
+
+// eslint-disable-next-line no-multi-spaces
+const { mongoURI } = require("./keys_dev");
+
+// find all in a colletion
+const findAllIn = (colName, query) => {
+  const client = new MongoClient(mongoURI, { useNewUrlParser: true });
```
Comment (jema28) [inline mongodb.js]: I'm not very experienced in mongodb but I don't think you need to connect for each function. @tawfiknasser have you used this?

File: mongodb.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+insertOneInto("users", { name: "majd" });
+
+// delete one into a colliction
+const deleteOneFrom = (colName, obj) => {
+  const client = new MongoClient(mongoURI, { useNewUrlParser: true });
+  client.connect(err => {
+    if (err) throw err;
+    const db = client.db("blooddonor");
+    db.collection(colName).deleteOne(obj, erro => {
+      if (err) throw erro;
```
Comment (jema28) [inline mongodb.js]: What's happening here? I would just console.log the error.

File: server/package.json [hunk 4]
```diff
@@ -21,7 +21,8 @@
     "compression": "^1.7.4",
     "express": "^4.16.4",
     "path": "^0.12.7",
-    "serve-favicon": "^2.5.0"
+    "serve-favicon": "^2.5.0",
```
Comment (jema28) [inline server/package.json]: Why are you adding serve favicon in this PR?

**Comment by jema28** [comment]
And please add a description @majdya!

File: server/package.json (hunk 4 above)
Comment (majdya) [reply server/package.json]: when resolving the conflict, it have been removed and I added it again.

**Pull request merged**